lib/Driver/ToolChains.cpp

```diff
@@ -104,11 +101,10 @@ addOutputsOfType(ArgStringList &Arguments,
 
 /// Handle arguments common to all invocations of the frontend (compilation,
 /// module-merging, LLDB's REPL, etc).
-static void addCommonFrontendArgs(const ToolChain &TC,
-                                  const OutputInfo &OI,
-                                  const CommandOutput &output,
-                                  const ArgList &inputArgs,
-                                  ArgStringList &arguments) {
+static void
+addCommonFrontendArgs(const ToolChain &TC, const OutputInfo &OI,
+                      const CommandOutput &output, const ArgList &inputArgs,
+                      ArgStringList &arguments) {
```

Comment:
Nitpick: Please don't reformat things if you're not actually changing them.

Reply:
How do I run git-clang-format so it only reformats the lines I changed?

Reply:
Fixed.

Reply:
I thought git-clang-format did that automatically. (That's what it says in its help output.) Is that not the case?

Reply:
I'll be watching it carefully!